packages/components/src/placeholder/README.md

```diff
@@ -11,3 +11,12 @@ const MyPlaceholder = () => (
 	/>
 );
 ```
+
+### Props
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`icon` | `string|element` | `undefined` | If provided, renders an icon next to the label.
```

Comment:
When I view this markdown file in GitHub, it's treating the pipe (`|`) here as a column separator. Not sure if that's a bug in GitHub's markdown parser. Might be worth checking what other markdown parsers do to see if we should avoid this syntax.